server/transport.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/go-kit/kit/endpoint"
 	kitlog "github.com/go-kit/kit/log"
 	kithttp "github.com/go-kit/kit/transport/http"
-	"golang.org/x/net/context"
+	"context"
```

Comment:
Don't make this change. Even though go-kit has switched to stdlib context, we can't yet. 
There are lots of places where we import `"golang.org/x/net/context"` in this project as well as other repositories in micromdm. It will take a bit of a coordinated effort to shift to stdlib context, and I'm not prepared to do it until go kit issues a 1.0.0 release in a few weeks. 


Instead, use `glide install` to download the correct dependencies to the `/vendor` folder and the project should compile. The correct version of go-kit is already pinned in the lock file. 

See https://github.com/Masterminds/glide